cyral/data_source_cyral_integration_logging.go

```diff
@@ -81,6 +77,7 @@ func dataSourceIntegrationLogs() *schema.Resource {
 					"SUMOLOGIC",
 					"FLUENTBIT",
 				}, false),
+				Default: "ANY",
 			},
 			"integration_list": {
```

Comment:
I know that some data sources use the list name in singular, but thinking better about that, I believe would be more correct to use plural here.
```suggestion
			"integrations_list": {
```

Reply:
@wcmjunior any thoughts about that? Maybe just `integrations` would be better?

Reply:
Absolutely.